src/parser/core/modules/CooldownDowntime.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import {getAction} from 'data/ACTIONS'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {ActionLink} from 'components/ui/DbLink'
+import {i18nMark, Trans} from '@lingui/react'
+
+export default class CooldownDowntime extends Module {
+	static handle = 'cooldowndowntime'
+	static title = 'CooldownDownTime'
+	static dependencies = [
+		'cooldowns',
+		'checklist',
+	]
+	static i18n_id = i18nMark('core.cooldowndowntime.title')
+
+	constructor(...args) {
+		super(...args)
+		//tracking the important™ CDs
+		this.trackedcds = []
+		//Default alloted time before a spell is held too long.
+		this.allowedDowntime = 0
```

Comment:
Perhaps if it is determined it is required later, currently doing that would break the checklist item which supposes a global allotment.  If ackwell pushes back on it I will, but I'm getting weary of having to handle for every potential possible corner case situation on initial push to expose functionality.  Until this push Cooldown didn't even support taking into consideration boss invuln time which completely invalided its output for anything other than CDs like Aetherflow.